Description: Always use getaddrinfo
 .
 The patch should probably be dropped and the configure check verified.
Origin: vendor
Forwarded: no
Last-Update: 2010-01-18

--- a/configure.in
+++ b/configure.in
@@ -692,50 +692,50 @@ PHP_CHECK_FUNC_LIB(nanosleep, rt)
 
 dnl Check for getaddrinfo, should be a better way, but...
 dnl Also check for working getaddrinfo
-AC_CACHE_CHECK([for getaddrinfo], ac_cv_func_getaddrinfo,
-[AC_TRY_LINK([#include <netdb.h>],
-                [struct addrinfo *g,h;g=&h;getaddrinfo("","",g,&g);], 
-  AC_TRY_RUN([
-#include <netdb.h>
-#include <sys/types.h>
-#ifndef AF_INET
-# include <sys/socket.h>
-#endif
-int main(void) {
-  struct addrinfo *ai, *pai, hints;
-
-  memset(&hints, 0, sizeof(hints));
-  hints.ai_flags = AI_NUMERICHOST;
-
-  if (getaddrinfo("127.0.0.1", 0, &hints, &ai) < 0) {
-    exit(1);
-  }
-
-  if (ai == 0) {
-    exit(1);
-  }
-
-  pai = ai;
-  
-  while (pai) {
-    if (pai->ai_family != AF_INET) {
-      /* 127.0.0.1/NUMERICHOST should only resolve ONE way */
-      exit(1);
-    }
-    if (pai->ai_addr->sa_family != AF_INET) {
-      /* 127.0.0.1/NUMERICHOST should only resolve ONE way */
-      exit(1);
-    }
-    pai = pai->ai_next;
-  }
-  freeaddrinfo(ai);
-  exit(0);
-}
-  ],ac_cv_func_getaddrinfo=yes, ac_cv_func_getaddrinfo=no, ac_cv_func_getaddrinfo=no),
-ac_cv_func_getaddrinfo=no)])
-if test "$ac_cv_func_getaddrinfo" = yes; then
+dnl AC_CACHE_CHECK([for getaddrinfo], ac_cv_func_getaddrinfo,
+dnl [AC_TRY_LINK([#include <netdb.h>],
+dnl                 [struct addrinfo *g,h;g=&h;getaddrinfo("","",g,&g);], 
+dnl   AC_TRY_RUN([
+dnl #include <netdb.h>
+dnl #include <sys/types.h>
+dnl #ifndef AF_INET
+dnl # include <sys/socket.h>
+dnl #endif
+dnl int main(void) {
+dnl   struct addrinfo *ai, *pai, hints;
+dnl 
+dnl   memset(&hints, 0, sizeof(hints));
+dnl   hints.ai_flags = AI_NUMERICHOST;
+dnl 
+dnl   if (getaddrinfo("127.0.0.1", 0, &hints, &ai) < 0) {
+dnl     exit(1);
+dnl   }
+dnl 
+dnl   if (ai == 0) {
+dnl     exit(1);
+dnl   }
+dnl 
+dnl   pai = ai;
+dnl   
+dnl   while (pai) {
+dnl     if (pai->ai_family != AF_INET) {
+dnl       /* 127.0.0.1/NUMERICHOST should only resolve ONE way */
+dnl       exit(1);
+dnl     }
+dnl     if (pai->ai_addr->sa_family != AF_INET) {
+dnl       /* 127.0.0.1/NUMERICHOST should only resolve ONE way */
+dnl       exit(1);
+dnl     }
+dnl     pai = pai->ai_next;
+dnl   }
+dnl   freeaddrinfo(ai);
+dnl   exit(0);
+dnl }
+dnl   ],ac_cv_func_getaddrinfo=yes, ac_cv_func_getaddrinfo=no, ac_cv_func_getaddrinfo=no),
+dnl ac_cv_func_getaddrinfo=no)])
+dnl if test "$ac_cv_func_getaddrinfo" = yes; then
   AC_DEFINE(HAVE_GETADDRINFO,1,[Define if you have the getaddrinfo function])
-fi
+dnl fi
 
 dnl Check for the __sync_fetch_and_add builtin
 AC_CACHE_CHECK([for __sync_fetch_and_add], ac_cv_func_sync_fetch_and_add,
